runtime/near-evm-runner/benches/benches/cryptozombies.rs

```diff
@@ -36,3 +38,45 @@ pub fn deploy_code(c: &mut Criterion) {
         })
     });
 }
+
+fn alice_account() -> AccountId {
+    "alice.near".to_string()
+}
+fn bob_account() -> AccountId {
+    "bob.near".to_string()
+}
+
+pub fn transfer_erc721(c: &mut Criterion) {
+    let mut ext = MockedExternal::new();
+    let bytes =
+        hex::decode(include_bytes!("../../tests/build/ZombieOwnership.bin").to_vec()).unwrap();
+    let outcome = evm_call(&mut ext, "alice", "deploy_code", bytes, false).0.unwrap();
+    let contract_id = address_from_arr(&outcome.return_data.as_value().unwrap());
+
+    let (input, _decoder) = cryptozombies::functions::create_random_zombie::call("test");
+    let args_create = encode_call_function_args(contract_id, input);
+
+    let alice_address = near_evm_runner::utils::near_account_id_to_evm_address(&alice_account());
+    let bob_address = near_evm_runner::utils::near_account_id_to_evm_address(&bob_account());
+
+    c.bench_function("transfer_erc721", |b| {
+        b.iter(|| {
+            // create zombie
+            let mut ext_copy = ext.clone();
+            evm_call(&mut ext_copy, "alice", "call", args_create.clone(), false).0.unwrap();
+
+            // transfer ERC-721 zombie token
```

Comment:
shall we bench create zombie and transfer token separately?

Reply:
Good point, since we measure creating a zombie already in another benchmark it probably makes sense to just measure the token transfer here. I'll change it accordingly.

Reply:
I changed it accordingly.